sencha-workspace/packages/slate-cbl/src/view/student/Dashboard.js

```diff
@@ -0,0 +1,154 @@
+/*jslint browser: true, undef: true *//*global Ext*/
+Ext.define('Slate.cbl.view.student.Dashboard', {
+    extend: 'Ext.Component',
+    xtype: 'slate-cbl-student-dashboard',
+    requires:[
+        'Slate.cbl.view.student.DashboardController',
+        'Slate.cbl.model.ContentArea',
+        'Slate.cbl.model.Competency',
+        'Slate.cbl.model.ContentArea',
+        'Slate.cbl.widget.Popover'
+    ],
+
+    controller: 'slate-cbl-student-dashboard',
+
+    config: {
+        contentArea: null,
+        popover: {
+            pointer: 'none'
+        }
+    },
+
+    competenciesTpl: [
+        '<tpl for="competencies">',
```

Comment:
@jmealo your robot formatter deleted the indentation these strings had for the HTML they contain